benchmark_orders_n1.php

```
@@ -0,0 +1,96 @@
+<?php
+// Mock functions to test logic and overhead
+function wc_get_orders($args) {
+    if ($args['return'] ?? '' === 'ids') {
+        return range(1, 100);
+    }
+
+    $orders = [];
+    for ($i = 1; $i <= 100; $i++) {
+        $orders[] = new MockOrder($i);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '10', column '25'). (undefined)

(MissingImport)